crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -709,7 +709,19 @@ impl CumulocityConverter {
         let command = command.to_owned();
         let payload = payload.to_string();
 
-        let mut logged = LoggedCommand::new(&command);
+        let mut args =
+            shell_words::split(&command).map_err(|e| CumulocityMapperError::ExecuteFailed {
+                error_message: e.to_string(),
+                command: command.to_string(),
+                operation_name: operation_name.to_string(),
+            })?;
+
+        let mut logged = LoggedCommand::new(args.remove(0));
+
+        if !args.is_empty() {
+            logged.args(&args);
+        }
+
         logged.arg(&payload);
```

Comment:
Beware, that the behavior is not obvious and has to be documented: the smart rest payload is *always* added as the very last argument.

The `command = "python /etc/tedge/operations/command.py"` actually leads to the execution of `python /etc/tedge/operations/command.py $SMART_REST_PAYLOAD`.